Editor/UI/Settings/AutopilotSettingsEditor.cs

```diff
@@ -185,7 +185,7 @@ internal void Launch(AutopilotState state)
             if (EditorApplication.isPlaying)
             {
                 state.launchFrom = LaunchType.EditorPlayMode;
-                Launcher.Run();
+                Launcher.Run().Forget();
```

Comment:
![ask](https://img.shields.io/badge/review-ask-blue.svg)

https://github.com/nowsprinting/Anjin/blob/fa6c0dc370825c3c967befb780257958e89a4d23/Runtime/LauncherFromTest.cs#L39
Should it be also forgotten?

Reply:
That's right.
But, `LaunchFromTest` will be significantly revised soon.
I don't want to make a difference now, so please leave it as it is.